sdk/schemaregistry/Azure.Data.SchemaRegistry/tests/SessionRecords/SchemaRegistryClientLiveTests/CanGetSchema(%Json%)Async.json

```diff
@@ -0,0 +1,107 @@
+{
+  "Entries": [
+    {
+      "RequestUri": "https://hmlam-cbn04-503.servicebus.windows.net/$schemaGroups/hmlam-json1/schemas/test-49275?api-version=2021-10",
```

Comment:
This test should be failing now in live mode. We will need to introduce a new api version in SchemaRegistryClientOptions.

Reply:
Yeah the live tests are all failing at the moment so I'm working on getting those working